src/ServiceControl/Notifications/Email/SendEmailNotificationHandler.cs

```diff
@@ -32,6 +32,8 @@ public async Task Handle(SendEmailNotification message, IMessageHandlerContext c
                 notifications = await manager.LoadSettings(cacheTimeout);
             }
 
+            logger.LogInformation("Processing email notification. Subject: {Subject}, Body: {Body}", message.Subject, message.Body);
```

Comment:
Given that we know these custom checks also get surfaced up to ServicePulse, I'm wondering if we want this to get info level at all.

Maybe it would be better to be a Debug level entry?